elections/steering/2025/README.md

```diff
@@ -0,0 +1,316 @@
+majority vote, and the criteria used during this process will be used to
+help refine further elections.
+
+If you otherwise qualify to vote but have not yet applied for Org Membership,
+then please [request an exception][voter exception form] (and please apply for
+Org Membership as well).
+
+Note that, if you are already eligible, instead of the voter exception form you
+will see "You are already eligible to vote in the election."
+
+Only contributions to projects and artifacts that fall under Steering
+Committee's governance will be considered for voter exception.
+
+Examples of contributions that would be considered:
+* Slack admins who are not active in GitHub
+* K8s Infra staff doing mostly support
+* Working Group leads without a lot of GitHub activity
+
+Examples of contributions that would NOT be considered:
+* Contributions to ecosystem projects and products
+* Organizing meetups or podcasts
+
+### Schedule
+
+<!-- While finalizing the dates in the schedule, ensure that:
+- The Steering Committee and candidate Q+A occurs at a public SC meeting
+  (usually a Wednesday).
+- Deadline to submit voter exception forms and request a
+  replacement ballot is ~3 days before voting closes.
+- Private announcement of results to SC members is at least ~2 days
+  before private announcement to all candidates.
+- The interval between private announcement to all candidates and the
+  public announcement is 24-48 hours.
+-->
+
+**The schedule below is an example and has not been updated for 2025**
+
+| Date                    | Event                                                                 |
+|:------------------------|:----------------------------------------------------------------------|
+| Tuesday, July 9         | Steering Committee selects Election Committee                         |
+| Thursday, August 1      | Announcement of Election and publication of voters.md                 |
+| Wednesday, August 21    | Steering Committee Q+A for the candidates (to be confirmed)           |
+| Saturday, August 24     | Candidate nominations due at the end of the day in AoE time           |
+| Sunday, August 25       | All candidate bios due at the end of the day in AoE time              |
+| Tuesday, August 27      | Election Begins                                                       |
+| Monday, September 23    | Deadline to submit voter exception requests                           |
+| Thursday, September 26  | Election Closes at the end of the day in AoE time                     |
+| Friday, September 27    | Private announcement of Results to SC members not up for election     |
+| Monday, September 30    | Private announcement of Results to all candidates                     |
+| Wednesday, October 2    | Public announcement of Results at Public Steering Committee Meeting   |
+| Wednesday, October 9    | Election Retro                                                        |
```

Comment:
I know you've copied last year, but it would be nice to update to match this years August dates :wink:

Reply:
The dates are decided by the officers, who can't do so until they are seated.